src/app/screens/Keysend/index.tsx

```diff
@@ -103,15 +72,25 @@ function Keysend(props: Props) {
             <>
               {destination && <PublisherCard title={destination} />}
 
-              <dl className="shadow bg-white dark:bg-surface-02dp pt-4 px-4 rounded-lg my-6 overflow-hidden">
-                {elements().map(([t, d], i) => (
-                  <Fragment key={`element-${i}`}>
-                    <dt className="text-sm font-semibold text-gray-500">{t}</dt>
-                    <dd className="text-sm mb-4 dark:text-white break-all">
-                      {d}
-                    </dd>
-                  </Fragment>
-                ))}
+              <dl className="text-sm shadow bg-white dark:bg-surface-02dp pt-4 px-4 rounded-lg my-6 overflow-hidden">
+                <dt className="font-medium text-gray-800 dark:text-white">
+                  Send payment to
```

Comment:
This is missing i18n. Could you open a separate PR for that or add it to #1460 ?